rskj-core/src/main/java/co/rsk/peg/bitcoin/UtxoUtils.java

```diff
@@ -0,0 +1,78 @@
+package co.rsk.peg.bitcoin;
+
+import co.rsk.bitcoinj.core.Coin;
+import co.rsk.bitcoinj.core.VarInt;
+import com.google.common.primitives.Bytes;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.spongycastle.util.encoders.Hex;
+
+public class UtxoUtils {
+    private UtxoUtils() {}
+
+    /**
+     * Decode a {@code byte[]} of encoded outpoint values.
+     *
+     * @param encodedOutpointValues
+     * @return {@code List<Coin>} the list of outpoint values decoded in the same ordered they were before encoding.
+     * Or an {@code Collections.EMPTY_LIST} when {@code encodedOutpointValues} is {@code null} or {@code empty byte[]}.
+     */
+    public static List<Coin> decodeOutpointValues(byte[] encodedOutpointValues){
+        if (encodedOutpointValues == null || encodedOutpointValues.length == 0) {
+            return Collections.emptyList();
```

Comment:
I really like how the method returns an empty list. Makes it very clean and easy to use :)

Reply:
Thanks! If there's no further comment about this feel free to mark this discussion as resolved whenever you ready :).